ncm-afsclt/src/main/perl/afsclt.pm

```diff
@@ -2,997 +2,281 @@
 # ${developer-info}
 # ${author-info}
 
-# Coding style: emulate <TAB> characters with 4 spaces, thanks!
-################################################################################
-
 package NCM::Component::afsclt;
 
-#
-# a few standard statements, mandatory for all components
-#
-
 use strict;
+use Readonly;
 use NCM::Component;
 use NCM::Check;
 use LWP::Simple;
+use CAF::FileWriter;
+use CAF::FileEditor;
+use CAF::Process;
 
 use vars qw(@ISA $EC);
 @ISA = qw(NCM::Component);
-$EC=LC::Exception::Context->new->will_store_all;
-
+$EC  = LC::Exception::Context->new->will_store_all;
 $NCM::Component::afsclt::NoActionSupported = 1;
```

Comment:
use
```perl
our $NoActionSupported = 1;
```

(i totally missed that before, i also find it "intersting" this was set before)